eos-migrate-chrome.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Replace eos-apps Chrome with Flathub Chrome
+ConditionPathExists=/var/lib/flatpak/app/com.google.Chrome/x86_64/eos3
+After=NetworkManager.service
+Wants=NetworkManager.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/lib/eos-boot-helper/eos-migrate-chrome
+Restart=no
+RemainAfterExit=yes
```

Comment:
I can never quite remember the semantics here, but I believe this means that, once the service is started once, then it will always be considered active, and so `systemctl start eos-migrate-chrome.service` will do nothing. If I'm right then I believe this is wrong. Consider the following sequence of events:

1. Computer becomes connected;
2. NM hook starts this service;
3. Computer becomes disconnected while downloading Flathub Chrome;
4. This service exits, but I believe it will still be considered to be running;
5. Now the computer goes offline and online again.

I believe in this case we would want the script to be started again, but due to `RemainAfterExit=yes` it won't be.